docs/academy/herocourse/module4.md

```diff
@@ -214,9 +206,11 @@ export async function handleStakingRewarded(event: SubstrateEvent): Promise<void
 
 
 
-#### Step 5: Build the project
+#### Step 5: Install Dependencies and Build the Project
+
+The next step is building the project. 
 
-Run the standard yarn install, codegen, build and docker-compose pull & docker-compose up commands.
+For that first. run the standard `yarn install`and then `yarn codegen`. After that run the `docker-compose pull & docker-compose up` command.  
```

Comment:
Unnecessary dot, I believe: For that first **.**

Reply:
Yes, that is a typo from my end. Thanks for noticing this.